packages/sdks/src/functions/evaluate/helpers.ts

```diff
@@ -52,6 +69,56 @@ export const getBuilderGlobals = (): BuilderGlobals => ({
   isBrowser: isBrowser(),
   isServer: !isBrowser(),
   getUserAttributes: () => getUserAttributes(),
+  track: (
+    eventName: string,
+    properties: Partial<EventProps & { apiHost?: string }> = {},
+    context?: any
+  ) => {
+    const builderContext = getGlobalBuilderContext();
+    _track({
+      type: eventName,
+      ...properties,
+      apiHost: builderContext?.apiHost,
+      apiKey: builderContext?.apiKey || '',
+      context,
+      canTrack: getDefaultCanTrack(properties.canTrack),
+    });
+  },
+  trackConversion: (
+    amount?: number,
+    contentId?: string,
+    variationId?: string,
+    customProperties?: any,
+    context?: any
+  ) => {
+    const meta = typeof contentId === 'object' ? contentId : customProperties;
+    let useContentId = typeof contentId === 'string' ? contentId : undefined;
+    const builderContext = getGlobalBuilderContext();
+
+    if (!useContentId && builderContext?.contentId) {
+      useContentId = builderContext.contentId;
+    }
+
+    let useVariationId = variationId;
+    if (!useVariationId && useContentId) {
+      useVariationId = getTestCookie(useContentId) || undefined;
+    }
```

Comment:
with all these fallbacks, i am thinking if we should add some e2e tests to make sure the content id will be correct:
- we can call track inside section models
- we can call track inside symbols (should pick the parent contentId)

Reply:
Added the e2e tests